pulp_container/app/tasks/sync_stages.py

```diff
@@ -443,12 +444,15 @@ def _get_content_data_blocking():
                 dl_res.artifact_attributes
             )
             content_data = json.loads(raw_data)
+            media_type = manifest_data.get("mediaType") or dl_res.headers.get("content-type")
+            if media_type is None:
+                raise ManifestInvalid(dl_res.artifact_attributes["sha256"])
             manifest = Manifest(
                 digest=manifest_data["digest"],
                 schema_version=2
```

Comment:
i think this information can be retrieved from `content_data`.Schemaversion is a  required field.